### PR TITLE
Service Mesh docs fixes during ROSA review

### PR DESCRIPTION
**Pull request by mburke5678**
Fixing various errors in the Applications docs as I find them during the ROSA content port. Mostly formatting, wording, and such.

Previews:
[Understanding Service Mesh](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-architecture) -- Updated topic map to match the assembly name. [Current docs](https://docs.openshift.com/container-platform/4.12/service_mesh/v2x/ossm-architecture.html).
[Configuring distributed tracing security for service mesh from the command line](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-reference-jaeger#distr-tracing-config-security-ossm-cli_jaeger-config-reference) -- Updated step 3 and 5 based on experienced. [Current docs](https://docs.openshift.com/container-platform/4.12/service_mesh/v2x/ossm-reference-jaeger.html#distr-tracing-config-security-ossm-cli_jaeger-config-reference).
[Configuring distributed tracing security for service mesh from the web console](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-reference-jaeger#distr-tracing-config-security-ossm-web_jaeger-config-reference) -- Formatting changes.
[Managing certificates with Elasticsearch](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-reference-jaeger#distr-tracing-manage-es-certificates_jaeger-config-reference) -- Removed version-specific prereqs.
[Adding or removing projects from the mesh using ServiceMeshMemberRoll resource with the CLI](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-mesh#ossm-add-project-member-roll-resource-cli_ossm-create-mesh) -- Formatting changes.
[Adding or removing projects from the mesh using the ServiceMeshMemberRoll resource with the web console](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-mesh#ossm-add-project-member-roll-resource-cli_ossm-create-mesh)  -- Formatting changes.
[Adding a project to the mesh using the ServiceMeshMember resource with the CLI](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-mesh#ossm-adding-project-using-smm-resource-cli_ossm-create-mesh) -- Moved verification steps into a new verification section.
[Adding a project to the mesh using the ServiceMeshMember resource with the web console](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-mesh#ossm-adding-project-using-smm-resource-console_ossm-create-mesh) -- Moved verification steps into a new verification section.
[Service Mesh architecture](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-architecture#ossm-architecture_ossm-architecture) -- Fixed typo in third paragraph.
[Configuring the Service Mesh Operator to run on infrastructure nodes](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/installing-ossm#ossm-config-operator-infrastructure-node_installing-ossm) -- Added metadata to code block in step 3.
[Deploying the Service Mesh control plane from the web console](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-smcp#ossm-control-plane-deploy-operatorhub_ossm-create-smcp) -- Split Step 6 into substeps. Moved verification steps into a new verification section. [Current docs](https://docs.openshift.com/container-platform/4.12/service_mesh/v2x/ossm-create-smcp.html#ossm-control-plane-deploy-operatorhub_ossm-create-smcp).
3scale configuration -- Assembly doesn't seem to appear in docs. Added metadata to code block.
[Configuring the control plane for cluster-wide deployment with the CLI](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-smcp#ossm-deploy-cluster-wide-control-plane-cli_ossm-create-smcp) -- Added new prereq (Needs QE), swapped period for a colon, moved verify steps to new section.
[Configuring the control plane for cluster-wide deployment with the web console](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-smcp#ossm-deploy-cluster-wide-control-plane-console_ossm-create-smcp) -- Added new prereq (Needs QE), swapped period for a colon, moved verify steps to new section.
[Deploying the distributed tracing platform](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-reference-jaeger#ossm-deploying-jaeger-production-min_jaeger-config-reference) -- Fixed [source] in Production distributed tracing platform (Jaeger) deployment, Control plane with linked Jaeger production CR, Control plane with linked Jaeger streaming CR examples. 
[Installing the Operators](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/installing-ossm#ossm-install-ossm-operator_installing-ossm) -- Split Step 6, bullet 3 into separate bullets. Moved verification steps into a new verification section.
[Operator overview](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/installing-ossm#ossm-installation-activities_installing-ossm) -- Added parenthesis to Kiali Operator (provided by Red Hat) for searchability in the console. 
[Creating the member roll from the web console](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-mesh#ossm-member-roll-create-console_ossm-create-mesh) -- Removed unneeded bolding on `one` in Step 8.
[Creating the member roll from the CLI](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-mesh#ossm-member-roll-create-cli_ossm-create-mesh) --  Removed unneeded bolding on `one` in Step 3.
[Adding or removing projects from the service mesh (v1)](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v1x/installing-ossm#ossm-member-roll-modify_installing-ossm-v1x) -- Removed unneeded bolding on `one` in second bullet
[Adding or removing projects from the member roll using the web console (v1)](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v1x/installing-ossm#ossm-member-roll-modify-console_installing-ossm-v1x) -- Removed unneeded bolding on `one` in Step 8.
[Adding or removing projects from the member roll using the CLI (v1)](https://67896--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v1x/installing-ossm#ossm-member-roll-modify-cli_installing-ossm-v1x)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/distr-tracing-config-security-ossm-cli.adoc [hunk 1]
```diff
@@ -35,7 +35,7 @@ $ oc project istio-system
 +
 [source,terminal]
 ----
-$ oc edit -n tracing-system -f jaeger.yaml
+$ oc edit Jaeger jaeger
 ----
```
Comment (mburke5678) [inline modules/distr-tracing-config-security-ossm-cli.adoc:39]: **QE:** Is my change the proper way to edit the Jaeger custom resource file? It is what worked for me. For ex, the tracing-system namespace doesn't exist.

File: modules/distr-tracing-config-security-ossm-cli.adoc [hunk 2]
```diff
@@ -78,16 +78,9 @@ spec:
...
-----
-$ oc apply -n tracing-system -f <jaeger.yaml>
-----
-+
 . Run the following command to watch the progress of the pod deployment:
 +
 [source,terminal]
 ----
-$ oc get pods -n tracing-system -w
+$ oc get pods -n istio-system
```
Comment (mburke5678) [inline modules/distr-tracing-config-security-ossm-cli.adoc:85]: **QE PTAL**
The tracing-system namespace doesn't exist. The other examples in the assembly suggest using `istio-system`.

File: modules/distr-tracing-config-storage.adoc [hunk 3]
```diff
@@ -700,8 +700,8 @@ spec:
 
 .Prerequisites
 
-* {product-title} 4.7
-* {logging-title} 5.2
+* The {SMProductName} Operator is installed.
+* The {logging-title} is installed with default configuration in your cluster.
 * The Elasticsearch node and the Jaeger instances must be deployed in the same namespace.  For example, `tracing-system`.
```
Comment (mburke5678) [inline modules/distr-tracing-config-storage.adoc:705]: **QE PTAL**

File: modules/ossm-deploy-cluster-wide-control-plane-cli.adoc [hunk 4]
```diff
@@ -12,6 +12,7 @@ You can configure the `ServiceMeshControlPlane` resource for cluster-wide deploy
 
 * The {SMProductName} Operator is installed.
 * You have access to the OpenShift CLI (`oc`).
+* There is no other `ServiceMeshControlPlane` object in the cluster.
```
Comment (mburke5678) [inline modules/ossm-deploy-cluster-wide-control-plane-cli.adoc:15]: **QE PTAL**

File: modules/distr-tracing-config-security-ossm-cli.adoc [hunk 5]
```diff
@@ -78,16 +78,9 @@ spec:
       readOnly: true
 ----
 +
-. Run the following command to apply your changes, where <jaeger.yaml> is the name of your Jaeger custom resource.
-+
-[source,terminal]
-----
-$ oc apply -n tracing-system -f <jaeger.yaml>
-----
-+
```
Comment (mburke5678) [inline modules/distr-tracing-config-security-ossm-cli.adoc:87]: Step not needed.

**Comment by pbajjuri20** [comment]
@mburke5678 : I have started reviewing this from the ServiceMesh QE team. 

If, I have any questions, I will let you know. 

Thanks for the changes in the docs.

File: modules/ossm-deploy-cluster-wide-control-plane-cli.adoc (hunk 4 above)
Comment (pbajjuri20) [reply modules/ossm-deploy-cluster-wide-control-plane-cli.adoc:15]: Add this Parameter to SMCP:

mode: ClusterWide in SMCP yaml, then you can deploy the Cluster-wide 

This is the error:

```
Fix the following errors:
admission webhook "smcp.validation.maistra.io" denied the request: a cluster-scoped SMCP may only be created when no other SMCPs exist
```

If, we delete the existing SMCP, then we can create the Cluster-wide SMCP.

File: modules/ossm-deploy-cluster-wide-control-plane-console.adoc [hunk 6]
```diff
@@ -12,6 +12,7 @@ You can configure the `ServiceMeshControlPlane` resource for cluster-wide deploy
 
 * The {SMProductName} Operator is installed.
 * You are logged in using an account with the `cluster-admin` role, or if you use {product-dedicated} with the `dedicated-admin` role.
+* There is no other `ServiceMeshControlPlane` object in the cluster.
```
Comment (pbajjuri20) [inline modules/ossm-deploy-cluster-wide-control-plane-console.adoc:15]: This statement is not useful in this section. we can remove it.

**Review by pbajjuri20 — APPROVED** [review]

**Review by pbajjuri20 — COMMENTED** [review]
Please, resolve only these two changes.

File: modules/ossm-control-plane-cli.adoc [hunk 7]
```diff
@@ -1,6 +1,7 @@
-// This module is included in the following assemblies:
-//
-// * service_mesh/v2x/ossm-create-smcp.adoc
+////
+This module is included in the following assemblies:
+* service_mesh/v2x/ossm-create-smcp.adoc
+////
```
Comment (mburke5678) [inline modules/ossm-control-plane-cli.adoc:4]: **PEER REVIEWER**: This format seems to be more common in the Service Mesh docs.

Comment (GroceryBoyJr) [reply modules/ossm-control-plane-cli.adoc:4]: Luv ya man, meenit. :-D

File: modules/ossm-add-project-member-roll-resource-console.adoc [hunk 8]
```diff
@@ -33,7 +33,7 @@ The `ServiceMeshMemberRoll` resource is deleted when its corresponding `ServiceM
 
 . Click the YAML tab.
 
-. Modify the YAML to add projects as members (or delete them to remove existing members). You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+. Modify the YAML to add projects as members (or delete them to remove existing members). You can add any number of projects, but a project can only belong to one `ServiceMeshMemberRoll` resource.
```
Comment (GroceryBoyJr) [inline modules/ossm-add-project-member-roll-resource-console.adoc:36]: Someone got WILD with the word "one". :-D

File: _topic_maps/_topic_map.yml [hunk 9]
```diff
@@ -3655,7 +3655,7 @@ Topics:
     File: ossm-about
   - Name: Service Mesh 2.x release notes
     File: servicemesh-release-notes
-  - Name: Service Mesh architecture
+  - Name: Understanding Service Mesh
```
Comment (GroceryBoyJr) [inline _topic_maps/_topic_map.yml:3658]:
```suggestion
  - Name: Understanding Service Mesh
```

Comment (GroceryBoyJr) [reply _topic_maps/_topic_map.yml:3658]: This adds the title "Understanding Service Mesh" twice: One at the  "top level" menu item and one immediately underneath. The adjacent repetition looks a little odd. I'd suggest the 2nd level title be "A network of microservices", but honestly, I am not at all sure that's an improvement.

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM.

**Pull request merged**